articles/active-directory/hybrid/howto-troubleshoot-upn-changes.md

```diff
@@ -165,6 +165,17 @@ The user will need to [re-enroll](/windows/security/identity-protection/hello-fo
 Windows 7 and 8.1 devices are not affected by this issue after UPN changes.
 
 
+## Mobile Application Management (MAM) app protection policies known issues and workarounds
+
+**Known Issues**
+
+Your organization may use [MAM app protection policies](https://docs.microsoft.com/mem/intune/apps/app-protection-policy) to protect corporate data in apps on end users' devices.
+MAM app protection policies are currently not resiliant to UPN changes. UPN changes can break the connection between existing MAM enrollments and active users in MAM integrated applications, resulting in undefined behavior. 
```

Comment:
nit: should we say `undefined` or `unprotected` here?

Reply:
How about both!